packages/engine/lib/orchestrator/src/error.rs

```diff
@@ -0,0 +1,41 @@
+use std::fmt;
+
+use error::Report;
+
+pub type Result<T, E = OrchestratorError> = error::Result<T, E>;
+
+// TODO: Use proper context type
+#[derive(Debug)]
+pub enum OrchestratorError {
+    Unique(String),
+    Report(Report<()>),
```

Comment:
Appreciate we're getting rid of this soon so if this is temporary lets not spend too much time, however, why do we have a `Report` variant here? Shouldn't `OrchestratorError` be the thing _inside_ of the `Report`?